plugins/persist-sqlite/Makefile

```diff
@@ -23,6 +23,12 @@ else
 ALL_DEPS:=
 endif
 
+ifeq ($(WITH_CJSON),yes)
+ALL_DEPS:=$(ALL_DEPS)
+else
+ALL_DEPS:=
+endif
+
```

Comment:
these two cjson patches seem unrelated to building on aix, are they just fixes for "building without json support" ?